external-crates/move/crates/move-compiler/src/typing/core.rs

```diff
@@ -914,7 +1131,8 @@ pub fn make_function_type(
     let public_for_testing =
         public_testing_visibility(context.env, context.current_package, f, finfo.entry);
     let is_testing_context = context.is_testing_context();
-    match finfo.visibility {
+    let visibility = finfo.visibility;
+    match visibility {
```

Comment:
Why hoist this out?

Reply:
Will revert, left over from when I tried to type check macros before expanding